code/go/pkg/validator/validator_test.go

```diff
@@ -813,6 +814,41 @@ func TestValidateForbiddenDataStreamName(t *testing.T) {
 	}
 }
 
+func TestValidateAnyOfContents(t *testing.T) {
+	tests := map[string][]string{
+		"missing_required_files": {
+			"item [.keep] is not allowed in folder [../../../../test/packages/missing_required_files/agent/input]",
+			"item [.keep] is not allowed in folder [../../../../test/packages/missing_required_files/data_stream/foo/agent/stream]",
+			"item [.keep] is not allowed in folder [../../../../test/packages/missing_required_files/data_stream/foo/elasticsearch/ingest_pipeline]",
+			"item [.keep] is not allowed in folder [../../../../test/packages/missing_required_files/data_stream/foo/fields]",
```

Comment:
Are the `.keep` files necessary? Would the missing files be found if the directories would not exist?

Reply:
git does not commit empty folders otherwise, and for the test I feel is more useful that the paths exist

Reply:
Yes, I was wondering if we would be missing the detection of the missing files if the directory would not exist. But I think that for these cases we are setting `required: true` in the directories.